From: Gabriel Valcazar <gabriel.valcazar@digi.com>
Date: Fri, 20 Jul 2018 18:04:55 +0200
Subject: [PATCH] imx_2d_device_g2d: define a G2D_AMPHION_INTERLACED macro

Currently, this symbol is not present in the libg2d used by imx6 platforms, so
this is a workaround to get the package to compile.

https://jira.digi.com/browse/DEL-6078

Signed-off-by: Gabriel Valcazar <gabriel.valcazar@digi.com>
---
 libs/device-2d/imx_2d_device_g2d.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libs/device-2d/imx_2d_device_g2d.c b/libs/device-2d/imx_2d_device_g2d.c
index 56dc550..e5d58c3 100755
--- a/libs/device-2d/imx_2d_device_g2d.c
+++ b/libs/device-2d/imx_2d_device_g2d.c
@@ -27,6 +27,8 @@
 GST_DEBUG_CATEGORY_EXTERN (imx2ddevice_debug);
 #define GST_CAT_DEFAULT imx2ddevice_debug
 
+#define G2D_AMPHION_INTERLACED 0x10
+
 typedef struct _Imx2DDeviceG2d {
   gint capabilities;
   struct g2d_surfaceEx src;
